launch/cylinder_spawner.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
+<!-- Spawn a robot into Gazebo -->
+    <param name="cylinder_description" command="$(find xacro)/xacro.py $(find vito_description)/model/cylinder.urdf.xacro" />
```

Comment:
That is, one should be able to run the test without having the vito package in the catkin workspace.